src/Compiler/TypedTree/TypedTreeOps.fs

```diff
@@ -9215,6 +9215,57 @@ let reqTyForArgumentNullnessInference g actualTy reqTy =
         changeWithNullReqTyToVariable g reqTy       
     | _ -> reqTy
 
+
+let GetDisallowedNullness (g:TcGlobals) (ty:TType) =
```

Comment:
Could you please add a comment here which things do we consider here?

Reply:
Just realized the comment is there in the .fsi file, so that it is visible as a doc comment as well.